### DISCUSSION TITLE
ENTE Auth only imported 10 codes from Google Authenticator

### DISCUSSION THREAD

**Discussion by ankitsinghaniyaz** [ente/ente#3517] [Q&A]
I am trying to migrate from google authenticator but Ente only imports 10 accounts from the authenticator app.

**Comment by frznn** [comment]
I think it's not related to the number but to specific codes that fail be imported, for some reason.
In my case it imported 11 codes without even givin any error on the failed ones (which is quite dangerous, since I could have uninstalled the Google app and lost the missing codes)

**Comment by HoggetNZ** [comment]
> I am trying to migrate from google authenticator but Ente only imports 10 accounts from the authenticator app.

Same - had to do it in batches of 10. (android)

**Comment by neeraj-pilot** [comment]
This should be fixed in the next release by #13017, which collects all Google Authenticator QR batches before importing.
